### PR TITLE
(DOCS-8698) Add AWS account-level logs guide

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do? What is the motivation?
Add a new guide for setting up AWS account-level log subscriptions

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/logs/guide/aws-account-level-logs.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+2. Go to [CloudFormation][1] in the AWS console.
+3. Click **Create stack**.
+    - Select `With new resources (standard)`.
+4. Leave the options to **Choose an existing template** and **Amazon S3 URL** checked.
+5. In the **Amazon S3 URL** field, paste the URL of the CloudFormation template.
+6. Click **Next**.
+7. In the **Stack name** field, provide a descriptive name such as `datadog-account-level-logs-stack`.
+8. In the **ApiKey** field, paste a valid [Datadog API key][4] value.
+9. In the **Regions** field, enter a comma-separated list of regions to enable metric streaming.
```
Comment (klivan) [inline content/en/logs/guide/aws-account-level-logs.md:31]: I would change `regions` to `AWS Regions` so it's more clear what regions we're looking for.
Also, `enable metric streaming` should be something like `enable account-level logs subscription"

Comment (buraizu) [reply content/en/logs/guide/aws-account-level-logs.md]: Thanks for reviewing and catching that, updated in [c7373f1](https://github.com/DataDog/documentation/pull/25398/commits/c7373f188698e361872e2921903a94d1665d026e)

**Comment by urseberry** [comment]
Created https://datadoghq.atlassian.net/browse/DOCS-9079 for documentation review.

File: content/en/logs/guide/aws-account-level-logs.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
+---
+title: AWS Account-Level Log Subscriptions
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+---
+
+You can use account-level log subscriptions in your AWS environment to automatically forward all of your CloudWatch logs to Datadog. With an account-level log subscription, you don't need to manually configure log forwarding when you have a new log source, or when AWS releases a new service. You can also define your own selection criteria or filter pattern for granular control over which logs are forwarded.
```
Comment (estherk15) [inline content/en/logs/guide/aws-account-level-logs.md:9]:
```suggestion
## Overview

You can use account-level log subscriptions in your AWS environment to automatically forward all of your CloudWatch logs to Datadog. With an account-level log subscription, you don't need to manually configure log forwarding when you have a new log source, or when AWS releases a new service. You can also define your own selection criteria or filter pattern, for more control over which logs are forwarded.
```

File: content/en/logs/guide/aws-account-level-logs.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+
+## Create an account-level log subscription
+
+For the simplest setup, use [CloudFormation](#cloudformation-recommended) to create an Amazon Data Firehose and associated resources in each of your selected regions.
```
Comment (estherk15) [inline content/en/logs/guide/aws-account-level-logs.md:13]:
```suggestion
There are two ways to create an account-level log subscription, through [CloudFormation](#cloudformation-recommended) and through [manual setup](#manual). For the simplest setup, use CloudFormation to create an Amazon Data Firehose and associated resources in each of your selected regions.
```

File: content/en/logs/guide/aws-account-level-logs.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+**Note**: To exclude certain log groups from log forwarding, use the `--selection-criteria` option as outlined in the [command reference][103].
+
+[101]: /logs/guide/send-aws-services-logs-with-the-datadog-lambda-function/
+[102]: https://aws.amazon.com/cli/
+[103]: https://docs.aws.amazon.com/cli/latest/reference/logs/put-account-policy.html
+{{% /tab %}}
+{{% tab "Amazon Data Firehose" %}}
+
+#### Create S3 bucket and role for Amazon Data Firehose
```
Comment (estherk15) [inline content/en/logs/guide/aws-account-level-logs.md:79]:
```suggestion
#### Create an S3 bucket and a role for Amazon Data Firehose
```

File: content/en/logs/guide/aws-account-level-logs.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+```
+
+3. Create an IAM role, specifying the trust policy file:
+
+```bash
+aws iam create-role \
+  --role-name FirehosetoS3Role \
+  --assume-role-policy-document file://./TrustPolicyForFirehose.json
+```
+   **Note**: The returned **Role.Arn** value is used in a later step.
```
Comment (estherk15) [inline content/en/logs/guide/aws-account-level-logs.md:112]: Recommend moving up:
Create an IAM role, specifying the trust policy file. **Note**: The returned **Role.Arn** value is used in a later step.

**Review by estherk15 — APPROVED** [review]
Left some optional suggestions. Also, I think if you tab the code blocks under the numbered lists, it makes it easier to see the numbered workflow.

**Pull request merged**